app/components/build-message.js

```diff
@@ -9,129 +9,130 @@ const { escapeExpression: escape } = Ember.Handlebars.Utils;
 export default Component.extend({
   readableMessage: computed('message.code', 'message.args', function () {
     let code = this.get('message.code');
+    let key  = this.get('message.key');
     let args = this.get('message.args');
```

Comment:
This is totally optional, but I think we can reduce these three lines into one like this:

`const { code, key, args } = this.message;`

There's a couple of things going on here:
- I think we can use `const` rather than `let`, since we won't need to reassign these variables a new value
- In current versions of ember we can now generally use `this.xyz` rather than needing to use `this.get('xyz')`, just a slightly shorter way to do the same thing
- We can use destructuring to assign multiple local variables (code, key, args) from the same parent object

But yeah, this is a small optional thing, either way is fine.

Reply:
done in https://github.com/travis-ci/travis-web/commit/537c39e16409e256068999a3fe8361547cb31d7f

thank you for the great review, @sorta ❤️

Reply:
Awesome, it looks good to me :smile: :heart: Thanks!